Libraries/Core/Devtools/symbolicateStackTrace.js

```diff
@@ -18,27 +18,38 @@ const {fetch} = require('fetch');
 
 import type {StackFrame} from 'parseErrorStack';
 
+function isSourcedFromDisk(sourcePath: string): boolean {
+  return !/^http/.test(sourcePath) && /[\\/]/.test(sourcePath);
+}
+
 async function symbolicateStackTrace(stack: Array<StackFrame>): Promise<Array<StackFrame>> {
   const devServer = getDevServer();
   if (!devServer.bundleLoadedFromServer) {
     throw new Error('Bundle was not loaded from the packager');
   }
-  if (SourceCode.scriptURL) {
-    for (let i = 0; i < stack.length; ++i) {
+
+  let stackCopy = stack;
```

Comment:
Might be confusing to call this a copy when it's not a copy if the code below doesn't run. How about just assigning `stack = [...stack]` and then keeping the old code?